docs/content/docs/tutorials/rds-example.md

```diff
@@ -166,7 +172,7 @@ kubectl get dbinstance "${RDS_INSTANCE_NAME}" -o jsonpath='{.status.endpoint.add
 kubectl get dbinstance "${RDS_INSTANCE_NAME}" -o jsonpath='{.status.endpoint.port}'
 ```
 
-You can find the username in the `DBInstance` spec (`spec.masterUsername`) and the password in the Secret that is reference in the `DBInstance` spec (`spec.masterPassword.name`).
+The database username and password is in the Secret that is referenced in the `DBInstance` spec (`spec.masterPassword.name`). If you did not set the username in the Secret, you can find it in the `DBInstance` spec (`spec.masterUsername`).
```

Comment:
> If you did not set the username in the Secret, you can find it in the `DBInstance` spec (`spec.masterUsername`).

This line does not seem applicable anymore? "If you did not set the username" especially.

Reply:
I modified that from the original line. This is to cover the case where a reader decides not to add the username into the Secret.

Reply:
If they are following this tutorial, then it is guaranteed that their username would be in the secret, though? Isn't that the point of the change?

Reply:
Setting the username in the Secret is not a requirement, so I wanted to provide additional details on where the user could find it.

For brevity, I can remove the line.